css/at-rules.json

```
@@ -120,6 +120,20 @@
     "status": "standard",
     "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@counter-style"
   },
+  "@container": {
+    "syntax": "@container <container-condition># {\n  <block-contents>\n}",
```

```suggestion
    "syntax": "@container <container-condition># {\n  <block-contents>\n}",
```

the syntax need to add to syntaxes.json if it is not existed in that file

---

I think this is still outstanding, right? I don't get any matches on this branch for `@container` aside from ./css/at-rules.json:

```bash
grep -r "@container" ./
./css/at-rules.json:  "@container": {
./css/at-rules.json:    "syntax": "@container <container-condition># {\n  <group-rule-body>\n}",
./css/at-rules.json:    "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@container"
```

---

Added, but unsure about the following 3; https://github.com/mdn/data/pull/853/files#diff-bf23b9edf4e332a630c103de7fc52e4eecc7d5262a21ac5feb2cc04a76cf2f40R230-R238